<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(185, 185, 185);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>690</width>
      <height>534</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QVTKOpenGLWidget" name="qvtkWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>400</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>360</y>
      <width>34</width>
      <height>178</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="greenI">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 205, 50);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pinkI">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 105, 180);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="blueI">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(135, 206, 235);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="orangeI">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 165, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tealI">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 128, 128);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="whiteI">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="blackI">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Background</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>330</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Item Color</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>570</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="Camera">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Reset Camera</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>34</width>
      <height>178</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="greenB">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 205, 50);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pinkB">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 105, 180);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="blueB">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(135, 206, 235);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="orangeB">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 165, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tealB">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 128, 128);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="whiteB">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="blackB">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openButton"/>
    <addaction name="saveButton"/>
   </widget>
   <widget class="QMenu" name="menuPreloaded">
    <property name="title">
     <string>Preloaded</string>
    </property>
    <addaction name="disk"/>
    <addaction name="arrow"/>
    <addaction name="cone"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="actionClip_Filter"/>
    <addaction name="actionShrink_Filter"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPreloaded"/>
   <addaction name="menuFilter"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openButton"/>
   <addaction name="saveButton"/>
   <addaction name="disk"/>
   <addaction name="arrow"/>
   <addaction name="cone"/>
  </widget>
  <action name="openButton">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/file.png</normaloff>:/Icons/file.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="saveButton">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/save.png</normaloff>:/Icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/print.png</normaloff>:/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="disk">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/disk.dng</normaloff>:/Icons/disk.dng</iconset>
   </property>
   <property name="text">
    <string>Disk Item</string>
   </property>
  </action>
  <action name="arrow">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/arrow.dng</normaloff>:/Icons/arrow.dng</iconset>
   </property>
   <property name="text">
    <string>Arrow Item</string>
   </property>
  </action>
  <action name="cone">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/slz.dng</normaloff>:/Icons/slz.dng</iconset>
   </property>
   <property name="text">
    <string>Cone Item</string>
   </property>
  </action>
  <action name="actionClip_Filter">
   <property name="text">
    <string>Clip Filter</string>
   </property>
  </action>
  <action name="actionShrink_Filter">
   <property name="text">
    <string>Shrink Filter</string>
   </property>
  </action>
  <action name="actionDarkMood">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/dark.png</normaloff>:/Icons/dark.png</iconset>
   </property>
   <property name="text">
    <string>DarkMood</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
